app.py

```diff
@@ -1,7 +1,7 @@
-from PyPDF2 import PdfReader
 import gradio as gr
 import openai
 import os
+from PyPDF2 import PdfReader
```

Comment:
Don't change the order of imported package, make this PR for "add comments" only